sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -935,38 +1016,499 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
+      defaultMessageTimeToLive: "P1D",
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      maxDeliveryCount: 5,
+
+      autoDeleteOnIdle: "PT2H",
+      authorizationRules: [
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Send"]
+          },
+          keyName: "allClaims_v2",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        },
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Listen"]
+          },
+          keyName: "allClaims_v3",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        }
+      ],
+
+      maxSizeInMegabytes: 49152,
+
+      forwardDeadLetteredMessagesTo: undefined,
+      forwardTo: undefined,
+      path: undefined,
+      userMetadata: undefined,
+
+      messageCountDetails: undefined,
+
+      enableExpress: undefined,
+      entityAvailabilityStatus: undefined,
+      isAnonymousAccessible: undefined,
+      supportOrdering: undefined,
+      status: undefined,
+
+      requiresDuplicateDetection: true,
+      requiresSession: true,
+      enablePartitioning: true,
+      queueName: alwaysBeExistingQueue
+    }
+  }
+].forEach((testCase) => {
+  describe(`Queue update with differing options #RunInBrowser`, function(): void {
+    it(`${testCase.testCaseTitle}`, async () => {
+      try {
+        const response = await updateEntity(
+          EntityType.QUEUE,
+          alwaysBeExistingQueue,
+          undefined,
+          undefined,
+          true,
+          testCase.input
+        );
+
+        assert.deepEqualExcluding(response, testCase.output, [
+          "_response",
+          "createdAt",
+          "updatedAt",
+          "accessedAt"
+        ]);
+      } catch (err) {
+        checkForValidErrorScenario(err, testCase.output);
+      }
+    });
+  });
+});
+
+// Topic tests
+[
+  {
+    topicName: "alwaysBeExistingTopic1",
+    testCaseTitle: "Undefined topic options",
+    input: undefined,
+    output: {
+      testErrorMessage: `Parameter "topicOptions" must be an object of type "TopicOptions" and cannot be undefined or null.`
+    }
+  },
+  {
+    topicName: "alwaysBeExistingTopic1",
+    testCaseTitle: "all properties",
+    input: {
+      sizeInBytes: 200,
+      messageCount: 5,
+      subscriptionCount: 8,
+      maxDeliveryCount: 10,
+      // enableExpress: true,
+
+      // maxSubscriptionsPerTopic: 3,
+      // maxSqlFiltersPerTopic: 4,
+      // maxCorrelationFiltersPerTopic: 5,
+      // isExpress: true,
+      // enableSubscriptionPartitioning: true,
+      // filteringMessagesBeforePublishing: true,
+
+      // maxSizeInMegabytes: 2048, // For partitioned entities, this is 16384
```

Comment:
Should these lines be removed, or are they something we plan to add later? (If the latter, can you add a comment indicating these should be added in the future with a tracking issue?)

Reply:
Added

Reply:
Sorry, I don't see the comment pointing to the tracking issue.